src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java

```diff
@@ -86,9 +86,13 @@ public final class TestConstants {
 
     public static final String EMPTY_STRING = "";
 
-    public static final String TEST_DIVORCE_APPLICATION_SOLE_TEMPLATE_ID = "NFD_CP_Application_Sole_V5.docx";
+    public static final String TEST_DIVORCE_APPLICATION_SOLE_TEMPLATE_ID = "NFD_CP_Application_Sole_V6.docx";
```

Comment:
Think the file name will need to be changed to the new one we created

Reply:
corrected